frontend/src/metabase/query_builder/components/QuestionActions.unit.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen } from "__support__/ui";
+import { createMockState } from "metabase-types/store/mocks";
+import { createMockEntitiesState } from "__support__/store";
+import QuestionActions from "metabase/query_builder/components/QuestionActions";
+import { createMockCard, createMockNativeCard } from "metabase-types/api/mocks";
+import { getMetadata } from "metabase/selectors/metadata";
+import Question from "metabase-lib/Question";
+
+const TEST_STRUCTURED_CARD = createMockCard();
+const TEST_NATIVE_CARD = createMockNativeCard();
+
+const iconList = [
+  { label: "bookmark icon", tooltipText: "Bookmark" },
+  { label: "info icon", tooltipText: "More info" },
+  {
+    label: "Move, archive, and more...",
```

Comment:
I'd like to maintain some consistency for the labels here, maybe there's a better label that we can use